.github/CODEOWNERS

```diff
@@ -263,7 +265,7 @@
 
 
 # Global components and filters.
-/core/templates/components/angular-html-bind/ @srijanreddy98
+/core/templates/components/interaction-display/ @srijanreddy98
```

Comment:
any specific reason for calling it interaction-display? (Note: angular-html-bind used to display any HTML template irrespective of whether it's part of the interaction or not!)